### PR TITLE
Reducing log collection.

### PR DESCRIPTION
**Pull request by jackyycheng**

### CODE REVIEW

File: connection-service/new-admin-vm/sumo-admin-vm.json [hunk 1]
```diff
@@ -1,24 +1,6 @@
...
-        "automaticDateParsing":  true,
-        "multilineProcessingEnabled":  true,
-        "useAutolineMatching":  false,
-        "forceTimeZone":  false,
-        "timeZone":  "Etc/UTC",
-        "filters":  [
-                    ],
-        "cutoffTimestamp":  0,
-        "encoding":  "UTF-8",
-        "pathExpression":  "C:\\WindowsAzure\\**\\*.log",
```
Comment (bill-dall) [inline connection-service/new-admin-vm/sumo-admin-vm.json]: The DSC logs we use for debugging are somewhere under this path - how do we get those?

File: remote-workstations/new-agent-vm/sumo-agent-vm.json [hunk 2]
```diff
@@ -2,36 +2,36 @@
 "api.version": "v1",
 "sources": [
     {
-        "name":  "CAM_Deploy_RemoteWorkstation_Win_Teradici_Agent_Logs",
-        "category":  "CAM/Deploy/RemoteWorkstation/Win/Teradici/Agent/Logs",
+        "name":  "CAM_Deploy_ConnectionServer_Win_Azure_Logs",
```
Comment (bill-dall) [inline remote-workstations/new-agent-vm/sumo-agent-vm.json]: ?

File: remote-workstations/new-agent-vm/sumo-agent-vm.json [hunk 3]
```diff
@@ -2,36 +2,36 @@
...
+        "multilineProcessingEnabled":  false,
         "useAutolineMatching":  false,
         "forceTimeZone":  false,
         "timeZone":  "Etc/UTC",
         "filters":  [
                     ],
         "cutoffTimestamp":  0,
         "encoding":  "UTF-8",
-        "pathExpression":  "C:\\WindowsAzure\\Logs\\Plugins\\Microsoft.Powershell.DSC\\**\\*.log",
+        "pathExpression":  "C:\\ProgramData\\Teradici\\PCoIPAgent\\logs\\*.txt",
```
Comment (bill-dall) [inline remote-workstations/new-agent-vm/sumo-agent-vm.json]: Did we want to bring this one back?

File: remote-workstations/new-agent-vm/sumo-agent-vm.json [hunk 4]
```diff
@@ -2,36 +2,36 @@
...
+        "multilineProcessingEnabled":  true,
         "useAutolineMatching":  false,
         "forceTimeZone":  false,
         "timeZone":  "Etc/UTC",
         "filters":  [
                     ],
         "cutoffTimestamp":  0,
         "encoding":  "UTF-8",
-        "pathExpression":  "C:\\ProgramData\\Teradici\\PCoIPAgent\\logs\\*.txt",
+        "pathExpression":  "C:\\WindowsAzure\\**\\*.log",
```
Comment (bill-dall) [inline remote-workstations/new-agent-vm/sumo-agent-vm.json]: Did we want to bring this one back?

**Comment by jackyycheng** [comment]
@bill-dall Sorry my bad, it should be ok now.

**Comment by bill-dall** [comment]
Thanks Jacky. I think we still want to remove the section at line 22 of remote-workstations/new-agent-vm/sumo-agent-vm.json ???

**Comment by jackyycheng** [comment]
@bill-dall  Yes you are right. I removed that section. Thanks

**Pull request merged**